types/google-apps-script/google-apps-script.card-service.d.ts

```diff
@@ -853,6 +874,13 @@ declare namespace GoogleAppsScript {
       setWrapText(wrapText: boolean): DecoratedText;
     }
 
+    /**
+     * A horizontal divider.
+     */
+    interface Divider {
+      printJson(): string;
```

Comment:
It seems `Divider#printJson` does not exists. I test in GAS.

```js
function cardService() {
  const divider = CardService.newDivider();
  Logger.log(divider.printJson());
}
```

```console
11:21:20 お知らせ 実行開始
11:21:21 エラー TypeError: divider.printJson is not a function
cardService
```

Reply:
My bad. `Divider` is an empty interface and was causing a `no-empty-interface` lint error. I misunderstood how to fix it: I thought other empty interfaces were just adding `printJson()`, but it turns out they were not actually empty 🤦‍♂️. I disabled the tslint `no-empty-interface` for `Divider`, similar to the `Widget` interface and updated the PR.